### PR TITLE
secboot: version bump, unlock volume with key

### PR DESCRIPTION
**Pull request by bboozzoo**
A couple of patches split out from #9528

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Thanks, this looks good to me

**Review by pedronis — APPROVED** [review]
thanks

File: secboot/secboot_tpm.go [hunk 1]
```diff
@@ -278,6 +280,28 @@ func UnlockVolumeIfEncrypted(disk disks.Disk, name string, encryptionKeyDir stri
 	return filepath.Join("/dev/disk/by-partuuid", partUUID), false, nil
 }
 
+// UnlockVolumeUsingSealedKeyIfEncrypted unlocks an existing volume using the
```
Comment (cmatsuoka) [inline secboot/secboot_tpm.go:283]: This should be UnlockEncryptedVolumeUsingKey.

**Review by cmatsuoka — COMMENTED** [review]
Thanks Maciek.

File: secboot/secboot_tpm.go (hunk 1 above)
Comment (bboozzoo) [reply secboot/secboot_tpm.go:283]: Thanks for spotting, I'll do quick followup

Comment (bboozzoo) [reply secboot/secboot_tpm.go:283]: Addressed in #9564

**Pull request merged**